modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/components/FieldSelectModalComponent.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+export class FieldSelectModalComponent {
```

Comment:
Having a component in tests doesn't feel right, we are typically scoping implementation parts into pages. What about something like
```
/data-set-admin/pages/components/FieldsSelectModalPage.ts
```

Reply:
No problem